lib/lita/adapters/slack/im_mapping.rb

```diff
@@ -15,7 +15,9 @@ def add_mapping(im)
         end
 
         def add_mappings(ims)
-          ims.each { |im| add_mapping(im) }
+          unless ims.nil?
```

Comment:
Is the nil check needed because ims is always null now that rtm.connect does not return that info? If so, do we also need to find the correct method for looking that up so that the functionality works?

Is this functionality used for sending timesheet reminders? (I haven't gotten one from ein-staging nor ein since the day that Slack changed the response of rtm.start to match that of rtm.connect.)

Reply:
@lizcorgi This functionality is used when connecting to Slack. Right now, @ein crashes upon startup due to various objects being nil due to the API change. 

I added just a nil check to avoid making too many invasive changes. I can understand why it might make sense to clean all of this up to match the new API response, but I also didn't want to end up inadvertently introducing issues.